python/ray/_private/serialization.py

```diff
@@ -127,7 +136,7 @@ def actor_handle_reducer(obj):
             serialized, actor_handle_id, weak_ref = obj._serialization_helper()
             # Update ref counting for the actor handle
             if not weak_ref:
-                self.add_contained_object_ref(actor_handle_id)
+                self.add_contained_object_ref(actor_handle_id, True)
```

Comment:
why it's always True for this case

Reply:
it crashes so many tests now. And I think the leak is probably very minimal for actor handle. I will add comments.

Reply:
(it doesn't leak actual actors)